Source/Fuse.Elements/Caching/ElementAtlas.uno

```diff
@@ -167,8 +159,12 @@ namespace Fuse.Elements
 			_framebuffer.Unpin();
 		}
 
-		void FillFramebuffer(DrawContext dc, bool drawAll, Rect scissorRectInClipSpace)
+		void FillFramebuffer(DrawContext dc, framebuffer fb, bool drawAll, Rect scissorRectInClipSpace)
 		{
+			// Changing the framebuffer is expensive. This loop actually doesn't draw anything in
+			// common case. Don't push the render target unless we actually have to render something.
```

Comment:
I thought so too initially, but no. The logic here is actually that we don't paint elements until they are *visible in the viewport*, yet we reserve space in the atlas. So the atlas can be used even though it has several invalid elements.